docs/validators/quickstart.mdx

```diff
@@ -50,19 +50,29 @@ connect version
 ## Run Connect Sidecar
 
 <Tabs>
-    <Tab title="Cosmos SDK Blockchains">
+    <Tab title="General">
+        This tab provides general instructions for starting the Connect sidecar.
+        If you are running one of the chains listed in the tabs above, please refer to those instructions.
         <QuickStartSnippet />
     </Tab>
-    <Tab title="dYdX Blockchain">
+    <Tab title="dYdX">
         <QuickStartDYDXSnippet />
     </Tab>
-    <Tab title="Neutron Blockchain">
-        **The minimum required Connect version for the Neutron chain is `v1.0.3+`.**
+    <Tab title="Neutron">
+        **The required Connect version for the Neutron chain is `v1.0.12`.**
         <QuickStartSnippet />
     </Tab>
-    <Tab title="Stargaze Blockchain">
+    <Tab title="Stargaze">
         <QuickStartStargazeSnippet />
     </Tab>
+    <Tab title="Warden">
+        **The required Connect version for the Warden chain is `v1.0.10`.**
+        <QuickStartSnippet />
+    </Tab>
+    <Tab title="Initia">
+        **The required Connect version for the Initia chain is `v1.0.8`.**
```

Comment:
should we just bump these ?

Reply:
like in their codebases? i was just going off https://github.com/initia-labs/initia/blob/main/go.mod#L49 and  https://github.com/warden-protocol/wardenprotocol/blob/main/go.mod#L58

Reply:
i don't think so, since connect v2 is proto breaking as we changed all the endpoints to be `connect/` instead of `slinky/`.

Reply:
But could we have them use `v1.0.12`?  This version includes the blast radius reduction included

Reply:
probably, i'll run some tests and make sure it works on those chains

Reply:
updated